CMakeLists.txt

```diff
@@ -296,6 +298,11 @@ if(REMILL_ENABLE_TESTING)
     add_subdirectory(tests/Thumb)
   endif()
 
+  if(REMILL_ENABLE_TESTING_SLEIGH_PPC)
```

Comment:
Do we need to add anything to the Actions workflows to get this running in CI?

Reply:
nope, should work since I added it into the cmake